src/common.jl

```diff
@@ -4,7 +4,7 @@ function letters_only(text)
 end
 
 function rotateRight(arr, n)
-  # implementation of the Mathematica function RotateRight
+  # implementation of the Mathematica function RotateRight - or you could try circshift()?
```

Comment:
I only knew because I wrote a version of it as well before I found `circshift`... Mathematica spoils us with so many built-functions...